CloudWatch2S3.template

```diff
@@ -147,13 +147,17 @@ Outputs:
             - LogBucket
             - Arn
         - Ref: BucketName
+    Export:
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-BucketArn'
```

Comment:
Why is the bucket export needed?

Reply:
just for completeness but can remove if you prefer.

Reply:
Completeness is good. And we prefix with the stack name so it shouldn't conflict.